src/redux/slices/bracket/bracketSlice.ts

```diff
@@ -28,14 +28,22 @@ export const bracketSlice = createSlice({
       state.matches[action.payload].isStarted = true;
       return state;
     },
-    finishMatch: (state, action: PayloadAction<{ matchId: number }>) => {
+    finishMatch: (
+      state,
+      action: PayloadAction<{
+        matchId: number;
+        firstPlayerScore: number;
+        secondPlayerScore: number;
+      }>,
+    ) => {
       //	TODO: Check if match was initialized
```

Comment:
Is it enough to check if `isStarted is true`?

Reply:
We are not given a match here. Only matchId. This is a for a function to set scores to a match with a certain id